var/spack/repos/builtin/packages/drishti/package.py

```diff
@@ -14,17 +14,24 @@ class Drishti(PythonPackage):
 
     homepage = "https://github.com/hpc-io/drishti-io"
     git = "https://github.com/hpc-io/drishti-io"
-    pypi = "drishti-io/drishti-io-0.4.tar.gz"
+    pypi = "drishti-io/drishti-io-0.6.tar.gz"
 
     maintainers("jeanbez", "sbyna")
 
     version("master", branch="master")
 
+    version("0.6", sha256="f1883e48de4a863f284f9c54febd2b04d04456848aea3e34c355310a3336e3b8")
+    version("0.5", sha256="08ae82dfa82872cde1f0a219e979b9c10d6c4a01f762066ea864a400cc144d78")
     version("0.4", sha256="bbbb272b4f6f44ae762f6cba28a2c589e15608691c559af0cc2f552590335d7b")
 
-    depends_on("darshan-util", type=("run"))
+    depends_on("darshan-util@3.4:", type=("run", "test"))
 
-    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
According to setup.py, 3.6 is still supported, but Spack only supports 3.7+ anyway, so I would just remove this line

Reply:
Got it, removed!